src/app/component/layout/components/search-all-results/search-all-results.component.ts

```diff
@@ -48,13 +48,21 @@ export class SearchAllResultsComponent implements OnInit, OnDestroy {
 
   private getSearchData(data: SearchModel): void {
     this.getNews(data.ecoNews);
-    // tslint:disable-next-line:no-unused-expression
-    data.countOfResults ? this.itemsFound = data.countOfResults : null;
+    if (data.countOfResults) {
+      this.itemsFound = data.countOfResults;
+    } else {
+      this.itemsFound = null;
+    }
```

Comment:
using ternary was good, but wrong styled. should be rewrite ternary only, not change it on `if`:
```js
this.itemsFound = ...
```

Reply:
Fixed